specification/cognitiveservices/data-plane/TranslatorText/preview/v1.0-preview.1/TranslatorBatch.json

```diff
@@ -526,22 +574,70 @@
           {
             "in": "query",
             "name": "$top",
-            "description": "Take the $top entries in the collection\r\nWhen both $top and $skip are supplied, $skip is applied first",
+            "description": "$top indicates the total number of records the user wants to be returned across all pages.\r\n            \r\nClients MAY use $top and $skip query parameters to specify a number of results to return and an offset into the collection.\r\nWhen both $top and $skip are given by a client, the server SHOULD first apply $skip and then $top on the collection.\r\n            \r\nNote: If the server can't honor $top and/or $skip, the server MUST return an error to the client informing about it instead of just ignoring the query options.",
             "type": "integer",
             "format": "int32",
-            "default": 50,
-            "maximum": 100,
-            "minimum": 1
+            "maximum": 2147483647,
+            "minimum": 0
           },
           {
             "in": "query",
             "name": "$skip",
-            "description": "Skip the $skip entries in the collection\r\nWhen both $top and $skip are supplied, $skip is applied first",
+            "description": "$skip indicates the number of records to skip from the list of records held by the server based on the sorting method specified.  By default, we sort by descending start time.\r\n            \r\nClients MAY use $top and $skip query parameters to specify a number of results to return and an offset into the collection.\r\nWhen both $top and $skip are given by a client, the server SHOULD first apply $skip and then $top on the collection.\r\n            \r\nNote: If the server can't honor $top and/or $skip, the server MUST return an error to the client informing about it instead of just ignoring the query options.",
             "type": "integer",
             "format": "int32",
             "default": 0,
             "maximum": 2147483647,
             "minimum": 0
+          },
+          {
+            "in": "query",
```

Comment:
nit: consider creating [parameters](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.4/TextAnalytics.json#L2549) definitions so you can reuse the type across different endpoints

Reply:
hmm, need more work to make this happen in code.  We'll look into this post release.

Reply:
i'll open a task on our side to keep track of this.